models/pull.js

```diff
@@ -240,4 +241,22 @@ Pull.getFromDB = function(data, signatures, comments, commitStatus, labels) {
    return new Pull(pullData, signatures, comments, commitStatus, labels);
 };
 
+Pull.isPassing = function() {
+   var required_builds = config.repos.find(function(repo) {
+      if (repo.full_name === this.repo) {
+         return repo.required_builds;
+      }
+   }).required_builds;
+
+   var passing_builds = this.commitStatuses.reduce(function(passingAccum, build) {
```

Comment:
Instead of a reduce, this can just be a filter for statuses that have a `state` of "success"

Reply:
c5fad924260069eb57e256e87019118a8f49e48c